cmd/sign.go

```diff
@@ -53,18 +55,26 @@ var signCmd = &cobra.Command{
 			fmt.Println(err)
 		}
 		fmt.Println("\nReceived OpenID Scope retrieved for account:", email)
+
 		key, pub, err := keymgmt.GeneratePrivateKey("ecdsaP256")
 		if err != nil {
 			fmt.Println(err)
 		}
 
-		h := sha256.Sum256([]byte(email))
-		proof, err := ecdsa.SignASN1(rand.Reader, key.(*ecdsa.PrivateKey), h[:])
+		privKey := key.(*ecdsa.PrivateKey)
+		pubKey := pub.(*ecdsa.PublicKey)
+		pubBytes, err := x509.MarshalPKIXPublicKey(pubKey)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
should return here and above on 61

Reply:
I don't think anything we can return to anything as this is called from &cobra.Command, what would we return to, root.go?

Reply:
Ah, you can use the RunE method in cobra so you can return an error

Reply:
Got you, I will look to implement this.